uwp/AutoComplete/Getting-Started.md

```diff
@@ -54,4 +54,6 @@ Dim textBox As New SfTextBoxExt()
 
 {% endtabs %}
 
+N> You can refer to our [UWP AutoComplete feature tour](https://help.syncfusion.com/uwp/autocomplete/getting-started) page to know about its other groundbreaking feature representations. You can also explore our [UWP AutoComplete example](https://apps.microsoft.com/store/detail/syncfusion-essential-studio-for-uwp/9NBLGGH5WNGV) to understand how to present and manipulate data.
```

Comment:
Hi @Edithabich , update feature tour page link. you have added UG page here. 
<img width="645" alt="image" src="https://github.com/syncfusion-content/uwp-docs/assets/44014283/6d2b87f1-3b89-4ff7-bb30-9b695853a6ed">